openapi/paths/reports@revenue-audit.yaml

```
@@ -0,0 +1,53 @@
+servers:
+  - url: 'https://api-sandbox.rebilly.com/experimental/organizations/{organizationId}'
+    description: Sandbox server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+  - url: 'https://api.rebilly.com/experimental/organizations/{organizationId}'
+    description: Live server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+get:
+  x-products:
+    - Reports
+  tags:
+    - Reports
+  summary: Retrieve a revenue audit report
+  operationId: GetRevenueAuditReport
+  x-sdk-operation-name: getRevenueAudit
+  description: >
+    Retrieve a revenue audit report which contains detailed journal entries.
+    It's a further expansion of revenue waterfall and journal reports.
+    It is a collection, so parameters as filter, limit and offset are available to work with it.
```

nitpick: I think it would be good to link to related content

I added a link which describes revenue waterfalls. We don't have one for journal reports yet (in this cycle I think)

```suggestion
  description: |
    Retrieve a revenue audit report which contains detailed journal entries. 
    A revenue audit report is a combination of a granular 
    [revenue waterfall](https://www.rebilly.com/docs/data-tables/revenue-recognition/#revenue-waterfall) 
    and a journal report, which describes revenue per account. 
    Use`Pagination-Limit` and `Pagination-Offset` parameters to filter the report.
```